lib/bumper.js

```diff
@@ -12,14 +21,48 @@ const utils = require('./utils')
  * @class
  */
 class Bumper {
+  // ====================================================================================
+  // Public Methods
+  // ====================================================================================
 
   /**
-   * @param {Vcs} vcs - the vcs instance to use
-   * @param {Config} config - the bumper config object
+   * @param {Object} params - params obj
+   * @param {Config} params.config - the bumper config object
+   * @param {Vcs} params.vcs - the vcs instance to use
+   * @param {Ci} params.ci - the ci instance to use
    */
-  constructor (vcs, config) {
-    this.vcs = vcs
-    this.config = config
+  constructor (params) {
+    this.config = params.config
+    this.vcs = params.vcs
+    this.ci = params.ci
```

Comment:
B/c what I'd *really* like to do is just 
```js
{this.ci, this.config, this.vcs} = params
```
But even `node@5.x` doesn't support it yet. If there was anything else in `params` or if it were any bigger, I might do what you suggested.

I only very recently factored that into a `params` object vs. positional arguments b/c I changed the order of the positional arguments and missed a spot which broke it.